test.sh

```diff
@@ -7,6 +7,18 @@
 # VERSIONS - Must be set to a list with possible versions (subdirectories)
 
 set -e
+# "best-effort" cleanup of previous image
+function clean_image {
+  if test -f .image-id.raw; then
+      local previous_id
+      previous_id=$(cat .image-id.raw)
+      if test "$IMAGE_ID" != "$previous_id"; then
+          # Also remove squashed image since it will change anyway
+          docker rmi "$previous_id" "$(cat .image-id)" || :
+          rm -f ".image-id.raw" ".image-id" || :
+      fi
+  fi
+}
```

Comment:
Does this not duplicate https://github.com/sclorg/container-common-scripts/blob/master/clean.sh?

Reply:
yes. I did not source it from test.sh, yet ;)

Reply:
Its used via `make clean`

Reply:
The function is present here: https://github.com/sclorg/container-common-scripts/blob/master/build.sh#L67

Reply:
I have imported the function from `build.sh` script